src/modules/RefreshmentMap/index.tsx

```diff
@@ -34,6 +39,51 @@ export const RefreshmentMap: FC<RefreshmentMapPropType> = (pageProps) => {
   const activeHour = HOURS[activeHourKey]
 
   const hourKeys = Object.keys(HOURS) as HourType[]
+  const [poiTooltipContent, setPoiTooltipContent] = useState<Pick<
+    MapPoiTooltipType,
+    'title' | 'category' | 'info'
+  > | null>(null)
+
+  const [poiTooltipCoordinates, setPoiTooltipCoordinates] = useState<{
+    latitude: number
+    longitude: number
+  } | null>(null)
+
+  const handleHover = (e: MapEvent): void => {
+    interface MapFeature {
+      source: string
+      properties: {
+        name?: string
+        category?: string
+        info?: string
+      }
+      [key: string]: unknown
+    }
+    interface CustomMapEventType extends MapEvent {
+      features: MapFeature[]
```

Comment:
I'm not sure why these interfaces are defined within the function here. You might want to move those to the top of the file to help the reader focus on the functionality. Or is there another reason?

Reply:
There *was* a reason while working on this. The `features` in `MapEvent` were typed as `any[]`, so it took me quite some trial and error to understand how to make that typing more expressive for us. During that process it was easier to have the types closeby. 

I changed it now.